### PR TITLE
feat: choose the email you want to be displayed

### LINKED ISSUE #5774: [CAL-499] Choosing the email you want to be displayed as the organizer
**Problem**

* I have a personal [cal.com](http://cal.com) account for (eg [myname@gmail.com](mailto:myname@gmail.com)).
* I also want to use [cal.com](http://cal.com) for my work purposes. To do that I attached my corporate Gmail calendar (eg [myname@mycompany.com](mailto:myname@mycompany.com)) to the existing personal [cal.com](http://cal.com) account. As a result now I get scheduled events in the right calendar ([myname@mycompany.com](mailto:myname@mycompany.com)) but as a participant I am displayed there with my personal email ([myname@gmail.com](mailto:myname@gmail.com))

**Desired outcome**

* Let the user choose on a per event basis which email address for them to appear as the organiser for.
* The list of choices should be pulled from their list of connected calendars.
* We would likely place this alongside the "Add to calendar" UI in the event type settings

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-499](https://linear.app/calcom/issue/CAL-499/choosing-the-email-you-want-to-be-displayed-as-the-organizer)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #5774 
This PR implements the following feedback

image
image

### Demo

Google Calendar Email:
https://www.loom.com/share/dc8c439a316f448ca2a84c497651554b

Apple Calendar:
https://www.loom.com/share/053f9a3f5fd54a98abfe4159f39a4b02

### CODE REVIEW

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Looking back at the issue. I noticed one of the desired outcomes was having a dropdown of available emails to display on the booking. I think we can add this, especially since not all calendars are associated with an email.

**Comment by gitstart-calcom** [comment]
Hi @joeauyeung Peer updated on the task https://github.com/calcom/cal.com/issues/5774#issuecomment-1505729370. From Peer's response, we thought we wouldn't need to create an additional dropdown, we did a dropdown before to select emails but we were asked to remove it

**Comment by joeauyeung** [comment]
@gitstart-calcom I talked with @ciaranha about this, and we think it's best to include a dropdown for the user to choose which email to display on the booking. It would also be nice to have a toggle similar to the video below that says that shows we are automatically choosing the email.

This should help handle calendars without emails associated with them and prevent any bugs in the future.

[

https://github.com/calcom/cal.com/assets/65426560/7c88c607-aa72-4bdb-b32f-abf95558494d

](url)

**Comment by joeauyeung** [comment]
Here are the designs as well: https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?type=design&node-id=4549-409104&mode=design

**Comment by gitstart-calcom** [comment]
Hi @joeauyeung we made the requested changes, please take a look 😄

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Awesome work with implementing the design I've tested this with the following calendars:
- GCal
- Outlook
- Radicale (CalDAV)
- Zoho (CalDAV)

I do have some feedback.
image
- This option should be opt in
- When it is selected it's still showing the account email

image
- The account email should be an option here

**Comment by gitstart-calcom** [comment]
Hey @joeauyeung We need some clarifications on the feedback:

- "This option should be opt in":
Does this mean the toggle should not be automatically selected ?

- "When it is selected it's still showing the account email":
The emails are blurred out, so it’s difficult to decipher what we should do here.

File: packages/trpc/server/routers/viewer/availability/calendarOverlay.handler.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { prisma } from "@calcom/prisma";
 import { getBusyCalendarTimes } from "@calcom/core/CalendarManager";
 import dayjs from "@calcom/dayjs";
+import { prisma } from "@calcom/prisma";
```
Comment (ThyMinimalDev) [inline packages/trpc/server/routers/viewer/availability/calendarOverlay.handler.ts:3]: formatted by pre commit hook

File: project.inlang.json [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source@1/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation@1/dist/index.js"
+  ],
+  "plugin.inlang.i18next": {
+    "pathPattern": "./apps/web/public/static/locales/{languageTag}/common.json"
+  },
+  "plugin.inlang.standardLintRules": {
+    "identicalPattern": "off"
+  }
 }
```
Comment (ThyMinimalDev) [inline project.inlang.json:56]: formatting done by pre-commit hook

File: packages/features/embed/lib/EmbedTabs.tsx [hunk 3]
```diff
@@ -118,7 +116,7 @@ export const tabs = [
       { calLink: string; embedType: EmbedType; previewState: PreviewState }
     >(function Preview({ calLink, embedType }, ref) {
       const bookerUrl = useBookerUrl();
-      const iframeSrc = `${EMBED_PREVIEW_HTML_URL}?embedType=${embedType}&calLink=${calLink}&embedLibUrl=${embedLibUrl}&bookerUrl=${bookerUrl}`
+      const iframeSrc = `${EMBED_PREVIEW_HTML_URL}?embedType=${embedType}&calLink=${calLink}&embedLibUrl=${embedLibUrl}&bookerUrl=${bookerUrl}`;
```
Comment (ThyMinimalDev) [inline packages/features/embed/lib/EmbedTabs.tsx:119]: formatting done by pre commit hook

**Comment by ThyMinimalDev** [comment]
not sure what is wrong with the commit tree of this branch, using git rebase to fix conflicts is not working

**Comment by PeerRich** [comment]
i dont think this PR is implemented correctly and not worth fixing

**Pull request closed without merging**